parsec/scheduling.c

```diff
@@ -484,7 +533,7 @@ int __parsec_context_wait( parsec_execution_stream_t* es )
     uint64_t misses_in_a_row;
     parsec_context_t* parsec_context = es->virtual_process->parsec_context;
     int32_t my_barrier_counter = parsec_context->__parsec_internal_finalization_counter;
-    parsec_task_t* task;
+    parsec_task_t* task, *last_local_task = NULL /* for debug purposes */;
```

Comment:
`last_local_task` seems unused

Reply:
it is temporary used [aka in draft mode] for debug purposes.